src/pathways/pathway/pathways.service.ts

```
@@ -282,6 +282,40 @@ export class PathwaysService {
+  /**
+   * True if any other VOLUNTEER pathway of the same subtype has an application
+   * window overlapping [openingDate, closingDate] — compares full ranges (not just
+   * "is anything currently open") so overlaps are caught whether the conflicting
+   * window is in the future or already backdated. NULL opening/closing bounds are
+   * treated as open-ended (no lower/upper bound).
+   */
+  private async hasSubtypeWindowConflict(
+    subtype: string,
+    openingDate: string | Date | null | undefined,
+    closingDate: string | Date | null | undefined,
+    excludePathwayId: string | null = null,
```

ESLint

[error] Delete `,`

(prettier/prettier)